src/test/java/org/springframework/data/redis/connection/jedis/JedisConvertersUnitTests.java

```diff
@@ -405,4 +407,117 @@ private Map<String, String> getRedisServerInfoMap(String name, int port) {
 		map.put("parallel-syncs", "1");
 		return map;
 	}
+
+	/**
+	 * Unit tests for {@link JedisConverters#toHGetExParams}.
+	 */
+	@Nested
+	class HGetExParamsTests { // GH-3211
```

Comment:
For some reason, we never got used to nested tests. Without having something tangible, it feels too verbose for too little value.